features/networking/multus.feature

```diff
@@ -866,3 +866,133 @@ Feature: Multus-CNI related scenarios
+      | {"spec":{"additionalNetworks": [{"name":"bridge-ipam-dhcp","namespace":"openshift-multus","rawCNIConfig":"{\"name\":\"bridge-ipam-dhcp\",\"cniVersion\":\"0.3.1\",\"type\":\"bridge\",\"master\":\"<%= cb.default_interface %>\",\"ipam\":{\"type\": \"dhcp\"}}","type":"Raw"}]}} |
+    #Cleanup for bringing CRD to original
+    Given I register clean-up steps:
+    """
+    as admin I successfully merge patch resource "networks.operator.openshift.io/cluster" with: 
+    | {"spec":{"additionalNetworks": null}} |
+    """
+    And admin ensures "bridge-ipam-dhcp" network_attachment_definition is deleted from the "openshift-multus" project after scenario
+    #Adding brige interface on target node
+    Given the bridge interface named "testbr1" is added to the "<%= cb.nodes[0].name %>" node
+    #Cleanup for deleting testbr1 interface
+    Given I register clean-up steps:
+    """
+    the bridge interface named "testbr1" is deleted from the "<%= cb.nodes[0].name %>" node
+    """
+    And a DHCP service is configured on the "<%= cb.nodes[0].name %>" node 
+    
+    #Cleanup for deconfiguring DHCP service on target node
+    Given I register clean-up steps:
+    """
+    a DHCP service is deconfigured on the "<%= cb.nodes[0].name %>" node
+    """
+    #Creating ipam type net-attach-def
+    Given I have a project
+    When I run oc create as admin over "https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/networking/multus-cni/NetworkAttachmentDefinitions/ipam-dhcp.yaml" replacing paths:
+      | ["metadata"]["name"]      | bridge-dhcp                                                                                                                                               |
+      | ["metadata"]["namespace"] | <%= project.name %>                                                                                                                                       |
+      | ["spec"]["config"]        | '{ "cniVersion": "0.3.0", "type": "bridge", "bridge": "testbr1", "hairpinMode": true, "master": "<%= cb.default_interface %>", "ipam": {"type": "dhcp"}}' |
+    Then the step should succeed
+    And admin ensures "bridge-dhcp" network_attachment_definition is deleted from the "<%= project.name %>" project after scenario
+
+    #Creating dhcp pod absorbing above net-attach-def
+    When I run oc create over "https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/networking/multus-cni/Pods/generic_multus_pod.yaml" replacing paths:
+      | ["metadata"]["namespace"]                                  | <%= project.name %>     |
+      | ["metadata"]["annotations"]["k8s.v1.cni.cncf.io/networks"] | bridge-dhcp             |
+      | ["spec"]["nodeName"]                                       | <%= cb.nodes[0].name %> |
+    Then the step should succeed
+    And the pod named "test-pod" becomes ready
+    When I execute on the pod:
+      | /usr/sbin/ip | a |
+    Then the output should contain "88.8.8"
```

Comment:
"88.8.8", is it a typo ?

Reply:
@liangxia no, it's correct, the last one is random , the ip is '88.8.8.x'

Reply:
ack

Reply:
@zhaozhanqi @liangxia I
Made """Given the bridge interface named""" definition more dynamic to take address range too from user. Its good for future use. Please help putting LGTM labels again :) Thanks